requirements/docs.txt

```diff
@@ -11,15 +11,15 @@ Sphinx==3.4.3
 docutils==0.14  # pyup: ignore
 
 sphinx_rtd_theme==0.5.1
-sphinx-tabs==1.3.0
+sphinx-tabs==2.0.1
```

Comment:
we can't update this because it breaks with our search extension, want to test if the version from master fixes this.